CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # CMakeLists to build the Spheral library.
 
 cmake_minimum_required(VERSION 3.10)
-project(spheral LANGUAGES CXX Fortran)
+project(spheral LANGUAGES C CXX Fortran)
```

Comment:
I don't know if this belongs here, Spheral doesn't build any C targets as part of it's code base.

Reply:
I think this may have come from building with Blast, where Spheral failed to build without the additional C language. I want to say axom or umpire threw the error there. If we don't need this for the standalone Spheral build, though, we can make a patch to change this line in Blast and remove the C here.

Reply:
Umpire would be the culprit of the two, they supply a C API. However we build our Umpire with the C interface enabled as well but it doesn't fail to build.